packages/typespec-vscode/rollup.config.ts

```diff
@@ -1,8 +1,13 @@
 import commonjs from "@rollup/plugin-commonjs";
 import resolve from "@rollup/plugin-node-resolve";
 import typescript from "@rollup/plugin-typescript";
+import path from "path";
 
 import { defineConfig } from "rollup";
+import { fileURLToPath } from "url";
+
+const curFile = fileURLToPath(import.meta.url);
+const curDir = path.dirname(curFile);
```

Comment:
I think now you can just do `import.meta.dirname` also curDir is not really accurate, `projDir`?

Reply:
updated to use import.meta.dirname. thx